README.md

```diff
@@ -333,3 +333,4 @@ Janus is thoroughly documented. You can find the current documentation, automati
 Any thought, feedback or (hopefully not!) insult is welcome!
 
 Developed by [@meetecho](https://github.com/meetecho)
+
```

Comment:
This doesn't seem to belong here.

Reply:
You marked this as resolved, but it isn't: I still see a new line here, which shouldn't be there.